rust/operator-binary/src/security/oidc.rs

```diff
@@ -25,15 +31,21 @@ pub enum Error {
     },
 
     #[snafu(display(
-        "found existing secret [{}/{}], but key {} is missing",
-        name,
-        namespace,
-        STACKABLE_OIDC_ADMIN_PASSWORD_KEY
+        "found existing admin password secret {secret:?}, but the key {STACKABLE_OIDC_ADMIN_PASSWORD_KEY} is missing",
     ))]
-    OidcAdminPasswordKeyMissing { name: String, namespace: String },
+    OidcAdminPasswordKeyMissing { secret: ObjectRef<Secret> },
+
+    #[snafu(display("invalid OIDC well known URL"))]
+    InvalidOidcWellKnownUrl {
+        source: stackable_operator::commons::authentication::oidc::Error,
+    },
```

Comment:
suggestion: Make it clear, that this refers to the well-known OIDC configuration URL.

```suggestion
    #[snafu(display("invalid well-known OIDC configuration URL"))]
    InvalidWellKnownConfigUrl {
        source: stackable_operator::commons::authentication::oidc::Error,
    },
```

_Note: This also needs updating of the context selector below_